core/variant/variant_construct.h

```diff
@@ -661,7 +661,7 @@ class VariantConstructNoArgsNil {
 		VariantInternal::clear(r_ret);
 	}
 	static void ptr_construct(void *base, const void **p_args) {
-		ERR_FAIL_MSG("can't ptrcall nil constructor");
+		ERR_FAIL_MSG("Cannot ptrcall nil constructor");
```

Comment:
Regex for MSG( starting with lower case, updated a few to fit other messages I found.

Reply:
What about exclamation points?

https://github.com/godotengine/godot/blob/da4f50851d15c073ca413196aa1501fb97108c07/scene/resources/skeleton_modification_2d_twoboneik.cpp#L286

Reply:
In most cases yeah, the exclamation mark might be overdoing it a bit. But that might imply further improving the phrasing for consistency with other similar errors, which may be outside the scope of this PR :)

Reply:
Understood. Leaving them alone for now.